R/main_script.R

```diff
@@ -1,3 +1,13 @@
+#' Remove trailing and preceding / from a given string
+#' @param input_string input string
+remove_trailing_and_leading_slashes <- function(input_string) {
+  # Remove leading slashes
+  input_string <- sub("^/*", "", input_string)
+  # Remove trailing slashes
+  input_string <- sub("/*$", "", input_string)
+  return(input_string)
+}
+
 #' Call an API once
```

Comment:
Best practice is to create documentation of a new function with devtools::document().
Also make sure to run devtools::test() to run tests.

Reply:
Function looks good!